### PR TITLE
Announcing Ticket Sales

### PR DESCRIPTION
**Pull request by gnunicorn**
- Tickets on Main page (not yet linked)
- Blog post with announcement
- Screenshot (as it looks on the main page) for Social Media

Not super psyched with the entire design (especially the last ticket), but good enough for now. Will more work on it during the weekend.

### CODE REVIEW

File: _posts/2016-07-28-ticket-sales-start-monday-august-1st.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+The news you've all been waiting for: you can finally secure your seats at RustFest in September.
+
+
+Starting Monday August 1st from 3pm (CEST) you can purchase supporter Tickets and View-Source-Combo-Tickets through our website. Exactly one week later you can buy the first batch of regular tickets. Thus giving you the chance to still buy one of the limited early-bird-combo-tickets before the deadline August 12th for a discount price.
+
+![](/assets/posts/tickets.png)
+
+
+Regular tickets will be **125€** (148,5€ incl. VAT), Supporter Tickets are priced at 250€. Until August 12th you can get the View Source Combo Ticket for 350€, which will be 450€ after that. A student discount of 20% is will be available on Regular Tickets.
```
Comment (badboy) [inline _posts/2016-07-28-ticket-sales-start-monday-august-1st.md]: >  student discount of 20% ~~is~~ will be available ...

remove the **is**

Comment (badboy) [inline _posts/2016-07-28-ticket-sales-start-monday-august-1st.md]: **148,50€** ? Prices are normally rounded to two digits, right?

**Comment by badboy** [comment]
I read the blog post. Small thing. Otherwise seems good.
I can check the design tonight

File: _posts/2016-07-28-ticket-sales-start-monday-august-1st.md (hunk 1 above)
Comment (gnunicorn) [inline _posts/2016-07-28-ticket-sales-start-monday-august-1st.md]: missing a `7` actually: **148,75**

Comment (badboy) [inline _posts/2016-07-28-ticket-sales-start-monday-august-1st.md]: I'm bad at math.

File: _posts/2016-07-28-ticket-sales-start-monday-august-1st.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+
+Starting Monday August 1st from 3pm (CEST) you can purchase supporter Tickets and View-Source-Combo-Tickets through our website. Exactly one week later you can buy the first batch of regular tickets. Thus giving you the chance to still buy one of the limited early-bird-combo-tickets before the deadline August 12th for a discount price.
+
+![](/assets/posts/tickets.png)
+
+
+Regular tickets will be **125€** (148,75€ incl. VAT), Supporter Tickets are priced at 250€. Until August 12th you can get the View Source Combo Ticket for 350€, which will be 450€ after that. A student discount of 20% will be available on Regular Tickets.
+
+We will distribute Regular Tickets in batches, opening at varying times of day giving an appropriate notice beforehand. So, if don't get any this time, another batch will come. Further more we will be distributing tickets through [diversitytickets.org](http://diversitytickets.org).
```
Comment (johannhof) [inline _posts/2016-07-28-ticket-sales-start-monday-august-1st.md]: - `Regular Tickets` -> `regular tickets`
- So, if **you** don't get any this time?
- `Further more` -> `Furthermore`

File: index.html [hunk 3]
```diff
@@ -124,6 +124,47 @@
...
+        </p>
+        <h4>available Aug 1st</h4>
+      </li>
+
+    </ul>
+    <p class="smallprint">
+      For a 20%-student discount please use the <em>STUDENT</em>-code at checkout. You will have to provide a valid international student ID as proof when picking up your conference badge.
+    </p>
+    <p class="smallprint">
+      Prices are advertised excluding VAT (where applicable). All sales are final. Tickets grant access to the conference, including food and drinks. The <a href="/code-of-conduct/">Code of Conduct</a> applies. Sales are awarded on a first-come-first-serve bases, but are sold in Batches. Registration for workshops will be handeled separately.
```
Comment (johannhof) [inline index.html]: `bases` -> `basis`
`handeled` -> `handled`

File: _posts/2016-07-28-ticket-sales-start-monday-august-1st.md (hunk 2 above)
Comment (gnunicorn) [inline _posts/2016-07-28-ticket-sales-start-monday-august-1st.md]: I understood "Regular Ticket", "Support Ticket" and "View Source Combo Ticket" (or just "Combo Ticket") as alone standing names. If we don't want to do that, then we should change that casing in the paragraph before, too, no?

@johannhof what do you think?

Comment (johannhof) [inline _posts/2016-07-28-ticket-sales-start-monday-august-1st.md]: I don't feel strongly about either choice, but yeah, it should be consistent. :)

File: _posts/2016-07-28-ticket-sales-start-monday-august-1st.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+The news you've all been waiting for: you can finally secure your seats at RustFest in September.
+
+
+Starting Monday August 1st from 3pm (CEST) you can purchase supporter Tickets and View-Source-Combo-Tickets through our website. Exactly one week later you can buy the first batch of regular tickets. Thus giving you the chance to still buy one of the limited early-bird-combo-tickets before the deadline August 12th for a discount price.
+
+![](/assets/posts/tickets.png)
+
+
+Regular tickets will be **125€** (148,75€ incl. VAT), Supporter Tickets are priced at 250€. Until August 12th you can get the View Source Combo Ticket for 350€, which will be 450€ after that. A student discount of 20% will be available on Regular Tickets.
```
Comment (gnunicorn) [inline _posts/2016-07-28-ticket-sales-start-monday-august-1st.md]: Comment from @flaki

> Do we want to explain in the blogpost in more detail what the supporter tickets are for, or should we leave that for the TITO page? (basically along the lines of “if your company pays your ticket, want to support our diversity outreach or generally help us make the conference more awesome you can choose this ticket type to support our efforts”)

**Pull request merged**